libraries/bsg_manycore_cuda.cpp

```diff
@@ -2053,9 +2053,11 @@ static int hb_mc_device_tiles_set_config_symbols (hb_mc_device_t *device,
 
 
 
-		// Set tile's tile group index  __bsg_tile_group_id_x/y symbol.
+		// Set tile's tile group index  __bsg_tile_group_id_x/y symbols.
+		// and tile's flat tile group id (tg_id_y * grid_dim_x + tg_id_x) 
 		hb_mc_idx_t tg_id_x = hb_mc_coordinate_get_x (tg_id); 
 		hb_mc_idx_t tg_id_y = hb_mc_coordinate_get_y (tg_id);
+                hb_mc_idx_t tg_id = tg_id_y * hb_mc_dimension_get_x(grid_dim) + tg_id_x;
```

Comment:
Can you fix the tabs vs spaces?

Reply:
thanks for pointing it out, since all lines in this file use tab, I changed this one to tab as well.